tools/serverpod_cli/lib/src/analyzer/entities/validation/restrictions.dart

```diff
@@ -105,14 +105,24 @@ class Restrictions {
   }
 
   List<SourceSpanException> validateTableIndexName(
-    dynamic content,
+    dynamic indexName,
     SourceSpan? span,
   ) {
-    if (content is! String ||
-        !StringValidators.isValidTableIndexName(content)) {
+    if (indexName is! String ||
+        !StringValidators.isValidTableIndexName(indexName)) {
+      return [
+        SourceSpanException(
```

Comment:
I think this may be a breaking change for some users. We should give a warning instead of an error here.

Reply:
We had this for 10 months we are safe :)